gate-core/src/test/groovy/com/netflix/spinnaker/gate/config/AuthConfigSpec.groovy

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2021 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.gate.config
+import com.netflix.spinnaker.fiat.shared.FiatClientConfigurationProperties
+import com.netflix.spinnaker.fiat.shared.FiatPermissionEvaluator
+import com.netflix.spinnaker.fiat.shared.FiatStatus
+import org.springframework.boot.autoconfigure.security.SecurityProperties
+import org.springframework.security.config.annotation.ObjectPostProcessor
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+import org.springframework.security.web.util.matcher.AnyRequestMatcher
+import spock.lang.Specification
+import java.util.stream.Collectors
+
+class AuthConfigTest extends Specification {
+  @SuppressWarnings("GroovyAccessibility")
```

Comment:
I'm not sure if this is kosher, but adding this annotation kept the compiler from complaining about the fact that I was accessing private properties in groovy. AFAICT, groovy doesn't actually have "true" private properties, so we can functionally access them. And because its in a test maybe its okay?